src/primitives/transaction.h

```diff
@@ -715,7 +658,6 @@ class CTransactionBase
     const std::vector<CTxOut>&        GetVout()       const {return vout;};
 
     virtual const std::vector<CTxScCreationOut>&      GetVscCcOut()   const = 0;
-    virtual const std::vector<CTxCertifierLockOut>&   GetVclCcOut()   const = 0;
```

Comment:
Just a minor comment: in transacition.h file, line 559 there is comment mentioning certLock, i.e. `CAmount certLockAmount;`
Remove it, so that the cmd `grep -inr "certlock"` returns nothing